mavros_msgs/msg/ComponentStatus.msg

```diff
@@ -0,0 +1,14 @@
+# Mavros message: COMPONENTSTATUS
+
+std_msgs/Header header
+
+uint8 cpu_usage               # percentage
+int8 temperature              # deg celsius
+uint32 ram_free               # MB
+uint32 ram_total              # MB
+uint32 storage_free           # MB
+uint32 storage_total          # MB 
+uint32 network_send_rate      # KB/s
+uint32 network_receive_rate   # KB/s
+uint32 time_boot_ms           # ms
```

Comment:
Usually we use header for message stamps.

Reply:
the field `time_boot_ms` is supposed to indicate the uptime of the component, so it is a bit more than a simple timestamp.

Reply:
Maybe uptime is a better name for the field then? time_boot_ms is used all across mavlink for packet timestamps.